### ISSUE TITLE
file extension of migration created by migration create should match template

### ISSUE THREAD

**Issue by jmjpro** [tj/node-migrate#110]
_Scenario_:
I created with migration_template.ts with contents
```
export const up = next => {
  next()
}

export const down = next => {
  next()
}
```
and ran `yarn migrate create --template-file ./migration_template.ts`.

_Expected_:
migration file in migrations directory is created with `.ts` extension

_Actual_:
migration file in migrations directory is created with `.js` extension

**Comment by LinusU** [comment]
Oh, this would be nice 👍 

PRs welcome!

**Comment by jmjpro** [comment]
Never mind I saw that it's possible already by passing --extension .ts when running the conmand. `extention` is a typo and should be `extension`. PR on the way...

**Comment by jmjpro** [comment]
https://github.com/tj/node-migrate/pull/111

**Comment by LinusU** [comment]
Still think it would be a nice feature if it defaulted to the template's extension?

**Comment by jmjpro** [comment]
yes. PR on the way...

**Comment by jmjpro** [comment]
I have the fix for this ready to go but first https://github.com/tj/node-migrate/issues/112 needs to be merged.

**Comment by wesleytodd** [comment]
Closed in #114
